docs/developers/json-rpc/geth_js_console.md

```diff
@@ -0,0 +1,256 @@
+- [Evmos (local) node](https://docs.evmos.org/developers/localnet/single_node.html) {prereq}
+
+### Installing Go-ethereum JavaScript Console
+
+Install the Go-ethereum JavaScript Console (`geth`) following the procedure corresponding to your OS in the [geth docs](https://geth.ethereum.org/docs/install-and-build/installing-geth).
+
+Check that the installation was successful by running the following command:
+
+```bash
+geth version
+```
+
+If everything went as expected, you should have an output similar to this:
+
+```bash
+Geth
+Version: 1.10.26-stable
+Git Commit: e5eb32acee19cc9fca6a03b10283b7484246b15a
+Architecture: amd64
+Go Version: go1.18.5
+Operating System: linux
+GOPATH=/home/tom/go
+GOROOT=/usr/local/go
+```
+
+## Install dependencies
+
+<!-- markdown-link-check-disable-next-line -->
+
+Make sure you have installed all the dependencies mentioned in the **[Pre-requisite Readings](#pre-requisite-readings)** section.
+
+## Run Evmos local node
+
+- Clone the [evmos repository](https://github.com/evmos/evmos) (if you haven’t already)
+- Run the `local_node.sh` script to start a local node
+
+```bash
+git clone https://github.com/evmos/evmos.git
+cd evmos
+./local_node.sh
+```
+
+## Attach geth JS console
+
+Wait a few seconds for the node to start up the JSON-RPC server.
+
+The local node has the HTTP-RPC server enabled and is listening at port 8545 by default. This is what we will connect to.
+
+Attach geth console to your node with the following command:
+
+```bash
+geth attach http://127.0.0.1:8545
+```
+
+If everything went alright, you should get a message similar to the one below.
+
+```bash
+Welcome to the Geth JavaScript console!
+
+instance: Version dev ()
+Compiled at  using Go go1.19.2 (amd64)
+coinbase: 0x7771fD5e52cf6A81B49d7EF40Bfc2bd0eA8A92E6
+at block: 975 (Fri Nov 25 2022 01:21:52 GMT-0300 (-03))
+ modules: debug:1.0 eth:1.0 net:1.0 personal:1.0 rpc:1.0 txpool:1.0 web3:1.0
+
+To exit, press ctrl-d or type exit
+> 
+```
+
+## Use JSON-RPC methods
+
+Now we can use all implemented JSON-RPC methods. Find an exhaustive list of the supported JSON-RPC methods on [Evmos docs](https://docs.evmos.org/developers/json-rpc/endpoints.html).
+
+Below are some examples of how to use the console.
+
+### Check current block height
+
+We can check the current block height of the chain.
+
+```bash
+eth.blockNumber
+```
+
+The result is printed on the next line. In the example below, we can see that we are at block height 1004:
+
+```bash
+> eth.blockNumber
+1003
+> eth.blockNumber
+1004
+```
+
+### Get accounts
+
+Get an array of the existing accounts in the keyring. To do so, use the following method:
+
+```bash
+eth.accounts
+```
+
+Example:
+
+```bash
+> eth.accounts
+["0xf0c3878dd8de6edc0702c06c2bb9a8e380397173", "0xfdd268dfeca95cff23ba385dec161defea031682", "0x35ab07f08f9af9166e9225b3407ad5e63756a084", "0x6a36c1efef7dd58981b3999217cdb3ae720cf330"]
+```
```

Comment:
I would prefer not to split the commands and examples up into two code blocks. I think it clutters up the document. Same for the rest of the examples.
```suggestion
```bash
> eth.accounts
["0xf0c3878dd8de6edc0702c06c2bb9a8e380397173", "0xfdd268dfeca95cff23ba385dec161defea031682", "0x35ab07f08f9af9166e9225b3407ad5e63756a084", "0x6a36c1efef7dd58981b3999217cdb3ae720cf330"]
```
```